### PR TITLE
model inconsistency

### PR DESCRIPTION
**Pull request by minoring**
The model is not consistent with the comment above.
I was wondering which one was incorrect; between the comment and the model.
As my assumption, I changed the comment.
(I changed and tested the model as the comment mentioned, and got about 50% accuracy on the validation set.)

### CODE REVIEW

File: site/en/tutorials/images/classification.ipynb [hunk 1]
```diff
@@ -1167,7 +1167,7 @@
         "id": "1Ba2LjtkVrXy"
```
Comment (yashk2810) [inline site/en/tutorials/images/classification.ipynb]: <p>How about this?</p><p> </p><p>"Applying dropout will randomly set 20% of the neurons to zero during each training epoch. This helps to avoid overfitting on the training dataset."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='86'/>

Comment (minoring) [reply site/en/tutorials/images/classification.ipynb]: Thank you for your kind review :)
I updated the comment as your advice which I think is more natural.
